src/main/java/com/openshift/restclient/model/IReplicationController.java

```diff
@@ -70,7 +71,7 @@
 	 * @param containerPorts  the container ports
 	 * @param volumes		 the set of emptyDir volumes to add to the config
 	 */
-	void addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
+	IContainer addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
```

Comment:
This breaks the API backward-compat. If this interface was added in 3.1, then it's fine, else, you should consider upversionning the rest client.
Also, it's probably a good idea to start adding @since tags in the javadoc